src/test/server/store-test.js

```diff
@@ -0,0 +1,201 @@
+const assert = require('assert');
+
+const usersData  = require('../../data/users.json');
+const UserAction = require('../../../_compiled_/actions/user-action').default;
+const UserInfo   = require('../../../_compiled_/stores/user-info').default;
+const UserStore  = require('../../../_compiled_/stores/user-store').default;
+const EVENT_TYPE = require('../../../_compiled_/stores/event-type').default;
+
+const TEST_USER = new UserInfo(
+    {
+        id:      11,
+        name:    'Ervin New',
+        phone:   '89500418181',
+        email:   'vika.chernookaya@mail.ru',
+        website: 'github.com'
+    }
+);
+
+const TEST_USER_EXISTING = new UserInfo(
+    {
+        id:      11,
+        name:    'Ervin Howell',
+        phone:   '0418181',
+        email:   'ervin@mail.ru',
+        website: 'github.com'
+    }
+);
+
+const SEARCH_USER_INFO = new UserInfo(
+    {
+        name: 'Ervin'
+    }
+);
+
+const SEARCH_USER_INFO_2 = new UserInfo(
```

Comment:
it's a bad name even for tests
Also we use only in one test so we can use implement it there